src/Form/Control/AddressLookup.php

```diff
@@ -16,19 +16,19 @@
 class AddressLookup extends Line
 {
     /** @var array An array of Google address_components to fill specific controls with. */
-    public $controlMap = [];
+    public array $controlMap = [];
```

Comment:
either remove `@var array ` phpdoc or revert

Reply:
Add typed array in PHPDoc, I think is better